verifiers/internal/gha/npm_sigstore_tuf.go

```diff
@@ -0,0 +1,114 @@
+package gha
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"path"
+	"runtime"
+	"time"
+
+	sigstoreTuf "github.com/sigstore/sigstore-go/pkg/tuf"
+)
+
+const (
```

Comment:
those need not be public, I think. You should be able to use lower case for first letter?
I's under internal directory so not a super big deal :)